### PR TITLE
added IVS_LoadStimSet to ivscc ipf as wrapper for NWB_loadallstimsets

### PR DESCRIPTION
**Pull request by aleonlein**

### CODE REVIEW

File: Packages/MIES/MIES_IVSCC.ipf [hunk 1]
```diff
@@ -367,6 +367,16 @@ Function IVS_RunGigOhmSealQC()
 	CtrlNamedBackground IVS_finishGigOhmSealQCCheck, start
 End
 
+// @brief Loads a single stimulus for the user when using the ZMQ Proxy
```
Comment (timjarsky) [inline Packages/MIES/MIES_IVSCC.ipf]: @aleonlein missing forward slash

Comment (t-b) [reply Packages/MIES/MIES_IVSCC.ipf]: @timjarsky Right. But unfortunately I was too eager to merge it. I'll fix that.

Comment (aleonlein) [reply Packages/MIES/MIES_IVSCC.ipf]: @timjarsky  good catch - I will fix and resubmit.

Comment (t-b) [reply Packages/MIES/MIES_IVSCC.ipf]: @aleonlein I've already fixed it in 2471.

**Pull request merged**